src/components/Avatar/Avatar.test.ts

```diff
@@ -1,6 +1,31 @@
 import { generateSnapshots } from '@chanzuckerberg/story-utils';
+import { getInitials } from './Avatar';
 import * as stories from './Avatar.stories';
 
 describe('<Avatar />', () => {
   generateSnapshots(stories);
+
+  // Testing handling of surrogate pairs, et al
+  // https://javascript.info/unicode#surrogate-pairs
+  describe('.getInitials', () => {
+    // ${'🧙‍♂️'}             | ${'🧙‍♂️'}
+    // ${'👦🏽🧶'}           | ${'👦🏽'}
+    // ${'☹️ 🙂'}          | ${'☹️🙂'}
```

Comment:
Are these commented out because they're composite emoji (or whatever the right terminology is... e.g. are made up of multiple graphemes)?

Reply:
Yah the wording around this is so hard :( But, correct; in those cases, there's 2+ parts to each. like the Wizard emoji is composed from the wizard AND the BMP for gender. when running the test on that one, it fails, but the expected and actual LOOK identical :'( 

The others behave incorrectly in different ways... i meant to take this off of review until i got a bit more time to see if there were any reasonable fixes to account for these cases, or maybe just discourage people from having emoji in their names and defer to the `displayName` prop